web/app/css/cta.css

```diff
@@ -1,5 +1,5 @@
 @import 'styles.css';
-@import 'font-awesome.min.css';
+@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css');
```

Comment:
we actually use font-awesome in a bunch of places (e.g. the tables), not just the CTA, so it does feel weird that it's in the `cta.css` file (it should have been moved a while back).

mind moving it to `styles.css` while you're here?

Reply:
I couldn't figure that out either. Happy to move it =)